### DISCUSSION TITLE
Fix typo in changelog for `v8.0.0`

### DISCUSSION THREAD

**Discussion by HonkingGoose** [actions/stale#989] [General]
There's a typo in the [changelog for `v8.0.0`](https://github.com/actions/stale/releases/tag/v8.0.0):

> New option labels-to-remove-when-stale enables users to specify list of **coma** delimited labels that will be removed when the issue or PR becomes stale by `@panticmilos` https://github.com/actions/stale/issues/770

The word `coma` should be changed to `comma`.

**Comment by MaksimZhukov** [comment]
Hello @HonkingGoose !
Thank you for noticing this! Fixed

**Comment by HonkingGoose** [comment]
Thank you for fixing it! ❤️
